### ISSUE TITLE
Asajj is allowed to shoot Dash in this scenario

### ISSUE THREAD

**Issue by boredompwndu** [baranidlo/bench#158]
![image](https://user-images.githubusercontent.com/8730661/41498989-5d7e18b4-7147-11e8-9187-d46227fb1)

This has happened for a few turns now. I've noticed it in other scenarios. I think its a rounding error involving multiple ships not moving

**Comment by Sleet01** [comment]
Looks like your link is malformed; do you have a link to the original picture?

**Comment by boredompwndu** [comment]
no. I can rebuild fairly easily. let me see how reproducible the situation is

**Comment by boredompwndu** [comment]
![image](https://user-images.githubusercontent.com/8730661/41631722-2e71878a-7404-11e8-9562-4588f37c2c70.png)

**Comment by boredompwndu** [comment]
![image](https://user-images.githubusercontent.com/8730661/41631738-3cc7aee0-7404-11e8-91c9-c22f7e231290.png)
